bin/halcyon-build

```diff
@@ -10,9 +10,17 @@ private_storage_setup() {
 [ -f ./.env ] && source ./.env
 
 if ! private_storage_setup; then
-  echo 'You must set the Halcyon variables for AWS credentials' >&2
-  echo 'https://halcyon.sh/reference/#private-storage-options' >&2
-  exit 1
+  cat >&2 <<EOF
```

Comment:
It seems likely to me that people will miss this message, as it will print just before Halcyon starts building everything.

What do you think about keeping this as an `exit 1`, but having an environment variable you can set to ignore it?